### ISSUE TITLE
Release 1.6.8 not available in Maven Central

### ISSUE THREAD

**Issue by lilvadim** [fvarrui/JavaPackager#301]
I'm submitting a…

- [ ] bug report
- [ ] feature request
- [x] other

**Short description of the issue/suggestion:**
Only 1.6.7 available.

**Steps to reproduce the issue/enhancement:**

1. Try to setup plugin as shown in README

**What is the expected behavior?**

Plugin applied

**What is the current behavior?**

```
> Could not resolve all files for configuration ':classpath'.
   > Could not find io.github.fvarrui:javapackager:1.6.8.
     Searched in the following locations:
       - https://repo.maven.apache.org/maven2/io/github/fvarrui/javapackager/1.6.8/javapackager-1.6.8.pom
       - https://plugins.gradle.org/m2/io/github/fvarrui/javapackager/1.6.8/javapackager-1.6.8.pom
```

**Please tell us about your environment:**

- JavaPackager version:
- OS version: macOS 13.2.1
- JDK version:
- Build tool: 
	- [ ] Maven 
	- [x] Gradle

**Comment by fvarrui** [comment]
@lilvadim try with `1.7.0|, please

**Comment by lilvadim** [comment]
Yes, it works with 1.7.0, but I tried 1.6.8 because I saw it is latest on
Releases page… It confuses a little.

**Comment by fvarrui** [comment]
Oops 😮! You are right. I've just fixed it as 1.6.8 was never released to Maven Central.
